files/conf/k8s-addons

```diff
@@ -134,12 +134,22 @@ do
 done
 
 {{ if .EnableAWSCNI -}}
-echo "Waiting for aws-node to be ready..."
-kubectl -n kube-system -l app.kubernetes.io/name=aws-node wait --for=condition=Ready --timeout=10m pods
+while
+    echo "Waiting for aws-node to be ready..."
+    kubectl -n kube-system -l k8s-app=aws-node wait --for=condition=Ready --timeout=1m pods
```

Comment:
I discovered this bug while testing. We were waiting on `app.kubernetes.io/name=aws-node` but `aws-node` only has a label of `k8s-app=aws-node`. It's not clear what the repercussions of this might have been. @giantswarm/team-firecracker-engineers could you check that this is still something we want to wait for? I believe it wasn't doing anything before (except possibly causing slower cluster creation).

Reply:
it was changed here: https://github.com/giantswarm/k8scloudconfig/commit/13efbd3fec01041ecdf95c6ba799071c3879d20d#diff-c60b5b7f6347684a39cf8f942fa109fa2706cee07062d20e900873dbd4356205


let's drop it, it was not doing anything for long time and it looks like having it here can cause problems where someone not from the AWS team changes something and it breaks :trollface:

Reply:
I can imagine this wait could improve AWS CNI stability during cluster creation and upgrades, but I'll go ahead and drop it since it apparently works without it (since it has been broken without us noticing).